packages/replayio/src/utils/installation/config.ts

```diff
@@ -18,8 +21,18 @@ const architecture: Architecture = process.arch.startsWith("arm") ? "arm" : "x86
 
 switch (process.platform) {
   case "darwin":
+    const homeDirectory = process.env.HOME || process.env.USERPROFILE;
+    assert(homeDirectory, "HOME or USERPROFILE environment variable must be set");
```

Comment:
Why not just use [`os.homedir()`](https://nodejs.org/api/os.html#oshomedir)?

Reply:
It's a copy-pasta from `getReplayPath`. Reading through `os.homedir` docs, it looks like it's a better fit - I'll switch both places to it